README.md

```diff
@@ -57,23 +57,46 @@ Please read [CONTRIBUTING.md](https://github.com/benawad/dogehouse/blob/staging/
 
 ## DogeHouse Desktop
 
-A desktop app built with [electron](https://www.electronjs.org/) is available for Windows, Mac, and Linux
+A desktop app built with [Electron](https://www.electronjs.org/) is available for Windows, Mac, and Linux.
 
 Download links are in [the releases section](https://github.com/benawad/dogehouse/releases/latest)
 
-Dogehouse is also available in the AUR
+Dogehouse is also available in the AUR...
 
 ```bash
 yay -S dogehouse
 ```
 
-And in an Ubuntu ppa
+...and in an Ubuntu PPA/self-hosted APT repoistory...
```

Comment:
change these lines to smtg like:
## Dogehouse Desktop
- Get the official electron build from here
- Get it from AUR
- Get ubuntu PPA
- Get snap
and then write the steps under each of them

Reply:
What specific version the deb/PPA have atleast minimum support (e.g. of minimum OS supported: Debian 8 / Ubuntu 16.06 LTS)? How about recommended OS version (eg of recommended OS supported: Debian 10 or whichever the latest release is tested / Ubuntu 20.04 LTS, can be the latest release after the current LTS)?

Reply:
for commit history it seems like `1.0.47` was where fixed linux builds.